.github/dependabot.yml

```
@@ -79,6 +79,11 @@ updates:
     allow:
       - dependency-type: direct
 
+    ignore:
+      # We need to wait for stylelint-config-gds to be ready for Stylelint 15
```

Shall we document the PR so anyone knows when to review this `ignore` rule?

```suggestion
      # Ignore major updates (causes `stylelint-config-gds` peer dependency conflict)
      # See pull request: https://github.com/alphagov/stylelint-config-gds/pull/32
```